src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/AllowedCharactersBitmap.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Yeah, that was what I wanted to say. It's just a visualization issue, it's not recorded in the repo and tools let you pick the specific percentage when showing history.